proposals/dex.md

```diff
@@ -0,0 +1,74 @@
+# Dex - Sandbox
+
+**Name of project:**
+
+Dex
+
+**Description:**
+
+Dex is an identity service that uses OpenID Connect to drive authentication for other apps.
+
+Dex acts as a portal to other identity providers through "connectors." This lets dex defer authentication to LDAP
+servers, SAML providers, or established identity providers like GitHub, Google, and Active Directory. Clients write
+their authentication logic once to talk to dex, then dex handles the protocols for a given backend.
+
+**Statement on alignment with CNCF mission:**
+
+Dex's main production use is as an auth-N addon in for Kubernetes. Dex runs natively on top of any Kubernetes cluster
+and can drive API server authentication through the OpenID Connect plugin. Clients, such as kubectl, can act on behalf
+users who can login to the cluster through any identity provider dex supports.
+
+Dex has been used by numerous companies who would like for it to find a neutral home where they can actively
+contribute to the long term viability of the project.
+
+**Sponsors from TOC:**
+
+TBD
+
+**Preferred maturity level:**
+
+Sandbox
+
+**License:**
+
+Apache2 + DCO
+
+**Source control:**
+
+Github (https://github.com/dexidp/dex)
+
+**Initial Committers:**
+
+Maintainers:
+
+- Joel Speed (Red Hat)
+- Stephan Renatus (Chef)
+- Nándor Krácser (Banzai Cloud)
+- Márk Sági-Kazár (Banzai Cloud)
+- Janos Matyas (Banzai Cloud)
+- Rithu John (Lyft)
+- Vijay Katam (VMware)
+- Mo Khan (VMware)
```

Comment:
@justaugustus Why have you added these names here?  

Why aren't these names in the project's official maintainers list ( https://github.com/dexidp/dex/blob/master/MAINTAINERS ) or https://github.com/dexidp/dex/graphs/contributors

What kind of contribution they have made to the project?  How long were they contributors for?

What is your definition of "Initial Committers" and "Maintainers"?

Cc @pritianka @caniszczyk @lizrice

Reply:
I believe including me in this list was a misunderstanding.  I reviewed Dex's code as part of VMware's efforts to help with the CNCF donation.  I am not a maintainer of Dex.

Reply:
- Who had the misunderstanding?
- What is that misunderstanding? Is it a misunderstanding about the "Initial Committers" and "Maintainers" words?
- Which dex pull request have you reviewed? 
- If it's not the pull request, what kind of review was it? Who has requested this review and where could we find the results of that review?
- There are 132+ dex contributors, why other contributors' names weren't misunderstood?

Reply:
@justaugustus can probably answer these questions better, but as far as the maintainers list is considered (https://github.com/dexidp/dex/blob/master/MAINTAINERS), these are all official maintainers of the project and the list is up to date.  We can probably just use that as source of truth.